auth/store.go

```diff
@@ -76,10 +76,20 @@ type AuthStore interface {
 
 	// RoleGrant grants a permission to a role
 	RoleGrant(r *pb.AuthRoleGrantRequest) (*pb.AuthRoleGrantResponse, error)
+
+	// TokenToUserID gets a UserID from the given Token
+	TokenToUserID(token string) (string, bool)
```

Comment:
Let's use username throughout the codebase? (actually we probably need to make it user which includes username and version. changing password will cause an increase of version. token will be assigned with a specific user version.)